dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantVectorStoreRecordCollection.cs

```diff
@@ -492,8 +495,8 @@ public async IAsyncEnumerable<VectorSearchResult<TRecord>> VectorizedSearchAsync
                 query: query,
                 usingVector: vectorName,
                 filter: filter,
-                limit: (ulong)internalOptions.Limit,
-                offset: (ulong)internalOptions.Offset,
+                limit: (ulong)internalOptions.Top,
+                offset: (ulong)internalOptions.Skip,
```

Comment:
Old names are still used, maybe it's correct because they are used by underlying DB.

Reply:
This reflects the underlying datastore naming, so this is correct.